vars/sectionBuildAndTest.groovy

```diff
@@ -90,13 +90,21 @@ def call(params) {
             withAcrClient(subscription) {
 
               def acbTemplateFilePath = 'acb.tpl.yaml'
+              def dockerfileTest = 'Dockerfile_test'
+              def isOnMaster = new ProjectBranch(env.BRANCH_NAME).isMaster()
 
               pcr.callAround('dockerbuild') {
                 timeoutWithMsg(time: 15, unit: 'MINUTES', action: 'Docker build') {
                   def buildArgs = projectBranch.isPR() ? " --build-arg DEV_MODE=true" : ""
-                  fileExists(acbTemplateFilePath) ?
+                  if(fileExists(acbTemplateFilePath)) {
                     acr.runWithTemplate(acbTemplateFilePath, dockerImage)
-                    : acr.build(dockerImage, buildArgs)
+                  } else {
+                    acr.build(dockerImage, buildArgs)
+                  }
+                  if (isOnMaster && fileExists(dockerfileTest)) {
```

Comment:
same comment as on plum:
https://github.com/hmcts/cnp-plum-recipes-service/pull/61/files#r331888192

the story says teams don't have to add dockerfiles to their repo, happy to be convinced that what we put in the story is a bad idea...